modules/mainFunction.js

```diff
@@ -0,0 +1,110 @@
+import checkStatus from './statusChecker.js';
+
+class TaskList {
+  constructor() {
+    this.tasksContainer = document.querySelector('.tasks');
+    this.addBtn = document.querySelector('.enter');
+    this.addText = document.querySelector('#starter');
+    this.form = document.querySelector('form');
+    this.taskKey = 'key';
+    const storedTasks = JSON.parse(localStorage.getItem(this.taskKey));
+    this.tasksInfo = storedTasks || [];
+
+    this.form.addEventListener('submit', (e) => {
+      e.preventDefault();
+      this.add();
```

Comment:
An excellent job was done in the mainfunction.js. 👌 💯
It would be good if ES6 implementation is done. Take a look [here](https://www.w3schools.com/js/js_arrow_function.asp) at how the interpretation of `this`  has a different meaning in the regular function and arrow function of ES6.

Reply:
Utilized all functions as arrow functions